go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/kpango/glg v1.5.1
 	github.com/lucasb-eyer/go-colorful v1.0.3
 	github.com/pierrec/lz4/v3 v3.3.2
+	github.com/pkg/errors v0.8.1
```

Comment:
I could not understand why this package is added into `go.mod`...

Reply:
i don't know

Reply:
I think it is caused by the `defaultCheckFunc`
```golang
	defaultCheckFunc := func(w want, got [][]float32) error {
		if !reflect.DeepEqual(got, w.want) {
			return errors.Errorf("got = %v, want %v", got, w.want)
		}
		return nil
	}
```

Reply:
I did not realize this. thx

Reply:
@vankichi no, I think I made an mistake... seems it is not caused by the `checkfunc`...

Reply:
but it is no longer exists now, so it should be ok~